### DISCUSSION TITLE
Download Snaps with quote reply

### DISCUSSION THREAD

**Discussion by meenoo10** [rodit/SnapMod#277] [Ideas & Feature Suggestions]
Is it possible to find a solution to download the "video" snaps with quoting responses?
the quote position becomes blurred. And the video resolution changes

Is it possible to add an option to download the video with quotes and comments without deleting anything? Just like downloading pictures.

![2221](https://user-images.githubusercontent.com/108699196/197601713-94ebe60a-9847-4f06-ad9b-d9581ada0e66.PNG)

Thank you

**Comment by RevealedSoulEven** [comment]
The image isn't clear bro

**Reply by meenoo10** [reply]
![2221](https://user-images.githubusercontent.com/108699196/197673332-a25b2ba7-a6d5-4eea-88a4-78cc435b9b3c.PNG)

It is supposed to be here "quoted reply "
The video shows distortion in the same position

**Comment by GoatBan** [comment]
Its not possible quoted reply work like a stickers & Snapchat blur everything under a stickers/emoji, its a security feature I Guess

**Reply by meenoo10** [reply]
https://savesnap.chat/

This site downloads videos with replies quoted without blur
But the quality of the video is lower than if you downloaded it from the tool

**Reply by meenoo10** [reply]
I hope the tool will not hide the replies quoted while maintaining high quality for videos.

**Reply by GoatBan** [reply]
> https://savesnap.chat/
> 
> This site downloads videos with replies quoted without blur
> But the quality of the video is lower than if you downloaded it from the tool

I did try with Sn4pm0d and its working fine for me 

![Snapchat-2111010546.jpg](https://user-images.githubusercontent.com/105595673/198080757-c96a4560-e632-4512-8ddd-2ad32821d824.jpg)

**Reply by meenoo10** [reply]
Yes, with pictures.
Does it work fine with videos?

**Comment by RevealedSoulEven** [comment]
That's a bug in snapmod I guess.... Maybe will be fixed soon

**Reply by meenoo10** [reply]
There is no bug
Some of the previous download tools work the same way

**Comment by RevealedSoulEven** [comment]
It won't work for videos bro
